apps/web-giddh/src/app/settings/Taxes/setting.taxes.component.ts

```diff
@@ -168,7 +168,12 @@ export class SettingTaxesComponent implements OnInit {
     public userConfirmation(userResponse: boolean) {
         this.taxConfirmationModel.hide();
         if (userResponse) {
-            if (this.confirmationFor === 'delete') {
+            if (this.confirmationFor === 'delete' && this.newTaxObj.taxType === 'others') {
```

Comment:
add nulity check here for newTaxObj

Reply:
public newTaxObj: TaxResponse = new TaxResponse(); this is call object this won't be null in any case